javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -369,6 +369,11 @@ public static AnnotationMirror getSameIgnoringValues(
             new Comparator<AnnotationMirror>() {
                 @Override
                 public int compare(AnnotationMirror a1, AnnotationMirror a2) {
+                    // AnnotationMirror.toString() prints the elements of an annotation in the
```

Comment:
We don't currently have to do this comparison.
The order of elements only matter if the annotation is the same, but the elements are not (Otherwise, areSame would return true.)  It's probably a bug for the same annotation with different elements to be in an annotation set, so I don't think the order matters.